config/locales/en.yml

```
@@ -17,6 +17,7 @@ en:
     check_your_email: Check your email
     not_found: Page not found
     internal_server_error: Sorry, there is a problem with the service
+    unprocessable_entity: Sorry, an invalid request was made
```

```suggestion
    unprocessable_entity: Sorry, there is a problem with the service
```

Users shouldn't need to know about requests; from their perspective there's a problem.